lib/xlsx_reader/styles.ex

```diff
@@ -77,6 +77,10 @@ defmodule XlsxReader.Styles do
     {"0.0%", :percentage},
     {~r/\Add?\/mm?\/yy(?:yy)\z/, :date},
     {~r/\Add?\/mm?\/yy(?:yy) hh?:mm?\z/, :date_time},
+    {"yyyy-mm-dd", :date},
+    {"yyyy-mm-dd h:mm:ss", :date_time},
```

Comment:
I think than an ISO-compliant datetime hour component is zero-padded. We should perhaps use a regex and match `hh?` like above to accept some variants.

The separator between date and time could also be a `T` instead of space, a regex could also handle that.

Reply:
Thanks for the feedback!

Interestingly I actually tried a regex first but it didn't work for me — when I debugged it the format that was being returned was `yyyy-mm-dd h:mm:ss` so I figured that was the custom pattern to match on. I agree though, there's no harm in assuming some variants here — I'm also armed with a bit more knowledge now so I'll have another crack at this!

Actually, this raises a question: I noticed you are doing a compatibility test with a Google spreadsheet — are there potentially differences to consider between Excel, Google Sheets, Numbers, LibreOffice etc.?

Reply:
I've noticed differences in the file naming conventions, handling of empty cells and of course the infamous date system. The format is such a mess, I'm sure there are many more…

Reply:
Hi @xavier — I've regexed that datetime format (added an optional `T` and `Z`). I guess there might be other formats but hopefully this is a bit more robust at least :)